pycbc/__init__.py

```diff
@@ -45,7 +46,21 @@
 __version__ = pycbc_version
 
 
-def init_logging(verbose=False, format='%(asctime)s %(message)s'):
+_default_format = '%(asctime)s %(message)s'
+
+
+class LogFormatter(logging.Formatter):
+    converter = dt.fromtimestamp
+
+    def formatTime(self, record, datefmt=None):
+        ct = self.converter(record.created).astimezone()
+        t = ct.strftime("%Y-%m-%dT%H:%M:%S")
+        s = "%s.%03d" % (t, record.msecs)
```

Comment:
F-strings please :)
```suggestion
        s = f"{t}.{record.msecs:03d}"
```
Is `record.msecs` really an integer?

Reply:
I used this to set it: https://stackoverflow.com/questions/6290739/python-logging-use-milliseconds-in-time-format/7517430#7517430

And it looks like it based on:
https://docs.python.org/3/library/logging.html#logrecord-attributes

But apparently it is a float